target_s3_csv/s3.py

```diff
@@ -29,6 +29,7 @@ def create_client(config):
     aws_secret_access_key = config.get('aws_secret_access_key') or os.environ.get('AWS_SECRET_ACCESS_KEY')
     aws_session_token = config.get('aws_session_token') or os.environ.get('AWS_SESSION_TOKEN')
     aws_profile = config.get('aws_profile') or os.environ.get('AWS_PROFILE')
+    aws_endpoint_url = config.get('aws_endpoint_url') or os.environ.get('AWS_ENDPOINT_URL')
```

Comment:
I think it's better to rely on the config alone to get the endpoint url, rather than support the AWS_ENDPOINT_URL env var. This makes it consistent with the other plugin https://github.com/transferwise/pipelinewise-tap-s3-csv.

env vars in this case are only for sensitive data such as credentials.

Reply:
good point - I made the update